dts/bindings/sensor/we,wsen-hids-common.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2022 Würth Elektronik eiSos GmbH & Co. KG
+# SPDX-License-Identifier: Apache-2.0
+
+properties:
+    drdy-gpios:
+      type: phandle-array
+      required: false
+      description: |
+        Data-ready interrupt pin.
+
+        Interrupt is active high by default.
+
+    odr:
+      type: string
+      required: false
+      default: "1"
```

Comment:
You must explicitly document why you chose this default in the description.

Do **not** try to justify this via GitHub reply to this comment only: you **must** include the justification in the description in the binding.

See https://docs.zephyrproject.org/latest/build/dts/bindings.html#general-rules

Reply:
Ok. I have actually removed the default value.